lib/embulk/input/mixpanel.rb

```diff
@@ -151,12 +170,18 @@ def run
             page_builder.add(values)
           end
 
+          if ignored_record_count > 0
+            Embulk.logger.warn "Skipped already loaded #{ignored_record_count} records. These record times are older or equal than previous fetched record time (#{prev_latest_fetched_time} @ #{prev_latest_fetched_time_format})."
+          end
           break if preview?
         end
 
         page_builder.finish
 
-        task_report = {to_date: @dates.last || (Date.today - 1)}
+        task_report = {
+          latest_fetched_time: current_latest_fetched_time,
+          to_date: @dates.last || Date.today - 1,
```

Comment:
This is no problem.
It can be optimized a little, but it is an out of scope on this PR.